### PR TITLE
attempted to compartmentalize html sections in views

### PR DESCRIPTION
**Pull request by djoii**
has 3 sections title, list and the button
need to figure out how to customize the filter
the title got screwed up a bit

everything else seems to be working though

### CODE REVIEW

File: src/Inbox.js [hunk 1]
```diff
@@ -12,32 +12,43 @@ class Inbox extends Component {
...
             </div>
-            <div className="inbox-title-right-item" style={{paddingLeft: "1em", marginTop: "-.1em"}}>
-              <img src="/ellipsis.png" alt="" style={{width: "20px"}}/>
+            <div className="title-right-item" style={{paddingLeft: "1em", marginTop: "-.1em"}}>
+              <img src="/ellipsis.png" style={{width: "20px"}}/>
             </div>
           </div>
         </div>
-        <ul style={{marginLeft: "-1.70em"}}>
+    ;
```
Comment (neilfulwiler) [inline src/Inbox.js]: is this trailing ";" necessary

File: src/Today.js [hunk 2]
```diff
@@ -12,8 +12,11 @@ class Today extends Component {
           <span className="is-size-7" style={{paddingLeft: "10px", color: "#808080"}}>
             {now.format("ddd MMM D")}
           </span>
+          
+        </div>
+        <div style={{paddingTop: "1em"}}>
```
Comment (neilfulwiler) [inline src/Today.js]: just making things consistent?

File: src/Inbox.js [hunk 3]
```diff
@@ -12,32 +12,43 @@ class Inbox extends Component {
...
   render() {
     const now = moment();
-    return (
-      <div className="inbox-container">
-        <div className="title inbox-title">
-          <span className="is-size-4" style={{paddingTop: "6px"}}>Inbox</span>
-          <div className="inbox-title-right">
-            <div className="inbox-title-right-item" 
+
+    const view = "Inbox";
```
[the remark this answers is not in the document]
Comment (djoii) [reply src/Inbox.js]: so as we compartmentalize the views we also must compartmentalize the CSS?

[the remark this answers is not in the document]
Comment (djoii) [reply src/Inbox.js]: made changes to app.sass and pushed

File: src/Inbox.js (hunk 1 above)
Comment (djoii) [reply src/Inbox.js]: i guess not

File: src/App.sass [hunk 4]
```diff
@@ -222,74 +216,6 @@ $greylike: #fafafa
...
-.next7days-container
-  text-align: left
-
-.add-task-hover
-  color: #db4c3f
-
-.add-task-no-hover
-  color: #808080
-
-.add-task-icon
```
Comment (neilfulwiler) [inline src/App.sass]: ?

File: src/Today.js (hunk 2 above)
Comment (neilfulwiler) [reply src/Today.js]: wondering why you added this styling that wasn't there before

**Comment by neilfulwiler** [comment]
I think you removed some stuff that we still need. Do the tasks still look right?

File: src/Today.js (hunk 2 above)
Comment (djoii) [reply src/Today.js]: for consistency sake yes

File: src/App.sass (hunk 4 above)
Comment (djoii) [reply src/App.sass]: please be more descriptive?

**Comment by djoii** [comment]
i believe i only removed what was redundant, and i think the todos still look correct, however now that you  mention it, the add task button looks different, but is now consistent.

**Pull request merged**